elements/johnson-mercier.def

```diff
@@ -0,0 +1,47 @@
+name: Johnson-Mercier
+html-name: Johnson&ndash;Mercier
+alt-names:
+  - Johnson&ndash;Mercier&ndash;K&rcaron;&iacute;&zcaron;ek
+min-order: 1
+max-order: 1
+categories:
+  - matrix
+  - macro
+reference-elements:
+  - triangle
+  - tetrahedron
+ndofs:
+  triangle:
+    formula: 15
+  tetrahedron:
+    formula: 42
+dofs:
+  facets: integral moments of normal-normal and normal-tangent inner products with (lagrange,1)
+  cell: integral moments of d(d+1)/2 components with (lagrange,0)
+mapping: double contravariant Piola
+sobolev: H(div div)
+implementations:
+  fiat: JohnsonMercier
+examples:
+  - triangle,1
+  - tetrahedron,1
```

Comment:
Need to leave off the examples until Symfem knows how to make them for this element
```suggestion
```